### ISSUE TITLE
Enable AWX (Ansible Tower) Schedule

### ISSUE THREAD

**Issue by AdamBrousseau** [eclipse-openj9/openj9#4572]
Parent #4221 
Related #4441 

- [ ] Schedule
- [ ] All machines or subset?
   - [ ] xlinux
   - [ ] plinux
   - [ ] zlinux
   - [ ] aix
   - [ ] osx
   - [ ] windows
- [ ] Testing
- [ ] Doc
- [ ] Logs

cc @jdekonin

**Comment by AdamBrousseau** [comment]
Stewart and co. have been working on and testing the playbooks but are currently not ready to run them regularly on production machines. SeeAdoptOpenJDK/openjdk-infrastructure#695. I think we should soft-block this task on their work and keep reevaluating the status.

**Comment by smlambert** [comment]
Agree, though you could enable each platform as they stabilize, to get a feel for how it is working.  

I also would encourage scheduled AWX tower jobs from a Jenkins job, because then we have a single point of contact and viewing, for tools / automation (if we want to know if machines changed overnight, simply look at the tower job in Jenkins).
